Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/TypeBindingGenerator.cpp

```diff
@@ -668,6 +720,19 @@ void GenerateTypeBindingHeader(FCodeWriter& HeaderWriter, FString SchemaFilename
 				*CPPCommandClassName(Class, RPC->Function));
 		}
 	}
+
+	// Generate blueprint-defined struct structures
```

Comment:
struct structures?

Reply:
haha I meant to say "its structure info i.e. property list" will reword